_data-prepper/pipelines/pipelines.md

```diff
@@ -84,269 +88,8 @@ conditional-routing-sample-pipeline:
-    - otel_metrics_raw_processor:
-  sink:
-    - opensearch:
-      hosts: ["https://localhost:9200"]
-      username: admin
-      password: admin
-```
-
-### S3 log ingestion pipeline
-
-The following example demonstrates how to use the S3Source and Grok Processor plugins to process unstructured log data from [Amazon Simple Storage Service](https://aws.amazon.com/s3/) (Amazon S3). This example uses application load balancer logs. As the application load balancer writes logs to S3, S3 creates notifications in Amazon SQS. Data Prepper monitors those notifications and reads the S3 objects to get the log data and process it.
-
-```yml
-log-pipeline:
-  source:
-    s3:
-      notification_type: "sqs"
-      compression: "gzip"
-      codec:
-        newline:
-      sqs:
-        queue_url: "https://sqs.us-east-1.amazonaws.com/12345678910/ApplicationLoadBalancer"
-      aws:
-        region: "us-east-1"
-        sts_role_arn: "arn:aws:iam::12345678910:role/Data-Prepper"
-
-  processor:
-    - grok:
-        match:
-          message: ["%{DATA:type} %{TIMESTAMP_ISO8601:time} %{DATA:elb} %{DATA:client} %{DATA:target} %{BASE10NUM:request_processing_time} %{DATA:target_processing_time} %{BASE10NUM:response_processing_time} %{BASE10NUM:elb_status_code} %{DATA:target_status_code} %{BASE10NUM:received_bytes} %{BASE10NUM:sent_bytes} \"%{DATA:request}\" \"%{DATA:user_agent}\" %{DATA:ssl_cipher} %{DATA:ssl_protocol} %{DATA:target_group_arn} \"%{DATA:trace_id}\" \"%{DATA:domain_name}\" \"%{DATA:chosen_cert_arn}\" %{DATA:matched_rule_priority} %{TIMESTAMP_ISO8601:request_creation_time} \"%{DATA:actions_executed}\" \"%{DATA:redirect_url}\" \"%{DATA:error_reason}\" \"%{DATA:target_list}\" \"%{DATA:target_status_code_list}\" \"%{DATA:classification}\" \"%{DATA:classification_reason}"]
-    - grok:
-        match:
-          request: ["(%{NOTSPACE:http_method})? (%{NOTSPACE:http_uri})? (%{NOTSPACE:http_version})?"]
-    - grok:
-        match:
-          http_uri: ["(%{WORD:protocol})?(://)?(%{IPORHOST:domain})?(:)?(%{INT:http_port})?(%{GREEDYDATA:request_uri})?"]
-    - date:
-        from_time_received: true
-        destination: "@timestamp"
-
-
-  sink:
-    - opensearch:
-        hosts: [ "https://localhost:9200" ]
-        username: "admin"
-        password: "admin"
-        index: alb_logs
-```
-
-## Migrating from Logstash
-
-Data Prepper supports Logstash configuration files for a limited set of plugins. Simply use the logstash config to run Data Prepper.
-
-```bash
-docker run --name data-prepper \
-    -v /full/path/to/logstash.conf:/usr/share/data-prepper/pipelines/pipelines.conf \
-    opensearchproject/opensearch-data-prepper:latest
-```
-
-This feature is limited by feature parity of Data Prepper. As of Data Prepper 1.2 release, the following plugins from the Logstash configuration are supported:
-
-- HTTP Input plugin
-- Grok Filter plugin
-- Elasticsearch Output plugin
-- Amazon Elasticsearch Output plugin
-
-## Configure the Data Prepper server
```

Comment:
Deleted this example because it's redundant to https://opensearch.org/docs/latest/data-prepper/managing-data-prepper/configuring-data-prepper/